image_builder/google-cloud-sdk.repo

```diff
@@ -1,16 +1,7 @@
-[google-compute-engine]
-name=Google Compute Engine
-baseurl=https://packages.cloud.google.com/yum/repos/google-compute-engine-el8-x86_64-stable
+[google-cloud-cli]
```

Comment:
My next push will include a comment indicating where the contents of this file came from.